dynawo/CMakeLists.txt

```diff
@@ -381,7 +378,7 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/sources/config.h.in ${CMAKE_CURRENT_S
 #
 add_custom_target(CREATE_DDBDIR
   COMMAND ${CMAKE_COMMAND} -E make_directory ${ddbdir}
-  )
+)
 
 if(MSVC)
   set(runtime_PATH "PATH=${ADEPT_HOME}/bin;${Boost_INCLUDE_DIR}/../lib;${dlfcn-win32_INCLUDE_DIRS}/../bin;${INSTALL_OPENMODELICA}/bin;${SUITESPARSE_HOME}/bin;${SUNDIALS_HOME}/lib;${XercesC_INCLUDE_DIRS}/../bin;${LibIIDM_BIN_DIR};${LibXML_BIN_DIR};${libdir};$ENV{PATH}")
```

Comment:
SHouldn't we update the LD_LIBRARY_PATH with the new libraries?

Reply:
Maybe; I did not want to do it before it worked since we cannot test it now

Reply:
done